lib/robot-controller/robots.rb

```diff
@@ -1,87 +1,93 @@
 require 'yaml'
 
 #
-class RobotConfigParser
-  ROBOT_INSTANCE_MAX = 16
+module RobotController
+  #
+  class Parser
+    ROBOT_INSTANCE_MAX = 16
```

Comment:
Can this be a configuration parameter?  Add documentation comments on why this limit is fixed.